Applets/UI/MainScreen/CellView.swift

```diff
@@ -8,46 +8,60 @@
 import SwiftUI
 
 struct CellView: View {
-
+    
     @ObservedObject var model: Goal
-
+    
     var body: some View {
-            ZStack {
-                if model.isEnabled {
-                    if model.percentOfDone < 100 {
-                        HStack(alignment: .lastTextBaseline) {
-                            Text("\(model.percentOfDone)")
-                                .font(.system(size: 100).bold().weight(.black))
-                                .padding(-10)
-                            Text("%")
-                                .font(.system(size: 40).bold().weight(.black))
-                                .padding(5)
-                        }
-                        .frame(width: 370, height: 100, alignment: .trailing)
-                        .foregroundColor(.white.opacity(0.30))
-                    } else {
-                        Text("DONE")
-                            .font(.system(size: 120).bold().weight(.black))
-                            .foregroundColor(.white.opacity(0.30))
+        ZStack(alignment: .center) {
```

Comment:
What if the goal isn't available yet? Let's add Gray cover to show it unavailable. But we need new attribute to core data entity isValid?